utils.go

```diff
@@ -283,6 +283,17 @@ func parseDSNParams(cfg *config, params string) (err error) {
 					return fmt.Errorf("Invalid value / unknown config name: %s", value)
 				}
 			}
+		case "connattrs":
+			if cfg.connattrs == nil {
+				cfg.connattrs = make(map[string]string)
+			}
+			for _, conn_v := range strings.Split(strings.TrimSuffix(strings.TrimPrefix(value, "("),")"), ",") {
```

Comment:
`strings.Split(strings.Trim(value, "()"), ",")` is shorter and not much weaker - but why not just forbid parens altogether?